json-schema-validator-all/2.2.6.wso2v1/pom.xml

```diff
@@ -0,0 +1,142 @@
+                            com.github.fge.msgsimple.*;version="${version.msg.simple}",
+                            com.github.fge.uritemplate.*;version="${version.uri.template}",
+                        </Export-Package>
+                        <Import-Package>
+                            !com.github.fge.*,
+                            com.google.common.base; version="${version.com.google.common}",
+                            com.fasterxml.jackson.core; version="${version.com.fasterxml.jackson.core}",
+                            com.google.common.collect; version="${version.com.google.common}",
+                            com.fasterxml.jackson.databind; version="${version.jackson.databind}",
+                            com.fasterxml.jackson.databind.node; version="${version.jackson.databind}",
+                            com.google.common.io; version="${version.com.google.common}",
+                            org.joda.time.format; version="${version.org.joda.time}",
+                            org.joda.time; version="${version.org.joda.time}",
+                            javax.mail.internet; version="${version.javax.mail.internet}",
+                            com.google.common.cache; version="${version.com.google.common}",
+                            com.google.common.primitives; version="${version.com.google.common}",
+                            com.google.i18n.phonenumbers; version="${version.com.google.i18n.phonenumber}",
+                        </Import-Package>
+                        <Private-Package>
+                        </Private-Package>
+                        <Include-Resource>
+                            @json-schema-core-${version.json.schema.core}.jar!/draftv4/*,
+                            @json-schema-core-${version.json.schema.core}.jar!/draftv3/*
+                        </Include-Resource>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <version.json.schema.validator>2.2.6</version.json.schema.validator>
+        <version.json.schema.core>1.2.5</version.json.schema.core>
+        <version.uri.template>0.9</version.uri.template>
+        <version.msg.simple>1.1</version.msg.simple>
+        <version.btf>1.2</version.btf>
+        <version.jackson.coreutils>1.8</version.jackson.coreutils>
+
+        <!--OSGI version range-->
+        <version.com.google.common>[15.0.0,16.0.0)</version.com.google.common>
+        <version.javax.mail.internet>[1.4.0,1.5.0)</version.javax.mail.internet>
+        <version.org.joda.time>[1.6.0, 3.0.0)</version.org.joda.time>
+        <version.jackson.databind>[2.6.0, 3.0.0)</version.jackson.databind>
+        <version.com.fasterxml.jackson.core>[2.6.0, 3.0.0)</version.com.fasterxml.jackson.core>
+        <version.com.google.i18n.phonenumber>[6.0.0, 8.0.0)</version.com.google.i18n.phonenumber>
```

Comment:
When mentioning import package range,  it is better to mention the property as "*.imp.pkg.version.range". 
please refer: https://github.com/wso2/orbit/pull/190/files